docs/decisions/0002-no-longer-rely-on-portal-designer.rst

```diff
@@ -0,0 +1,71 @@
+========================================
+2. No longer rely on ``portal-designer``
+========================================
+
+******
+Status
+******
+
+Accepted
+
+*******
+Context
+*******
+
+The ``portal-designer`` (or ``designer`` for short) service is a Wagtail/Django-based content management system (CMS) that was created in order to provide basic configuration data on a site-by-site basis for the Programs and Enterprise learner portal frontend applications. 
+
+For example, an individual Enterprise Customer (e.g., "Pied Piper") would have an associated site and "Enterprise Page" created in ``designer`` in order to configure the learner portal for that Enterprise. The configuration data for an individual Enterprise Customer consists of:
+
+* Enterprise Customer uuid (from the ``EnterpriseCustomer`` model in ``edx-enterprise`` / LMS)
+* Enterprise Customer name
+* Learning coordinator email address (exposes to enterprise learners in the learner portal)
+* Branding configuration
+    * Logo image
+    * Banner colors (background/border)
+
+As mentioned above, ``designer`` is also used to configure the Programs learner portals for each Master's degree for a given university partner (i.e., site). However, the configuration data for Programs differs from the configuration data for an Enterprise. For example, the learner portal for a Program might be configured to show links to program-related documents, render static text in the sidebar of that program's learner portal, or use custom background images in the banner.
+
+Additionally, the ``designer`` service provides an API that is called during the build process for the Programs and Enterprise learner portals in order to gather the necessary configuration data for each Program or Enterprise learner portal.
+
+However, by configuring the Programs and Enterprise learner portals in the same ``designer`` service, we are introducing a bit of a tight coupling between Programs and Enterprise, despite the configuration data being different for each learner portal. Further, as time goes on, the configuration for Enterprise and Programs learner portals will likely continue to diverge.
```

Comment:
Mostly a nit: I would suggest rephrasing to use a different tense for `we are introducing a bit of a tight coupling between Programs and Enterprise` (past or future) as we are not currently planning on coupling them together by being in the same configuration service.